tests/copier_data/data1.yaml

```
@@ -5,11 +5,12 @@ is_open_source: true
 description: Doing amazing things
 install_claude_cli: false
 ssh_port_number: 12345
+pull_from_ecr: false
 
 
 
 
```

YAMLlint (1.38.0)

[error] too many blank lines (4 > 2)

(empty-lines)